src/frontends/pytorch/src/op/convolution.cpp

```diff
@@ -11,42 +11,85 @@ namespace frontend {
 namespace pytorch {
 namespace op {
 
+Output<ov::Node> reshape_bias(NodeContext& context, Output<ov::Node> bias, Output<ngraph::Node> conv) {
+    auto conv_shape = context.mark_node(std::make_shared<opset8::ShapeOf>(conv));
+    auto conv_rank = context.mark_node(std::make_shared<opset8::ShapeOf>(conv_shape));
+    auto one_const = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {1}));
+    auto two_const = context.mark_node(opset8::Constant::create(element::i64, Shape{1}, {2}));
+    auto tail_shape_rank = context.mark_node(std::make_shared<opset8::Subtract>(conv_rank, two_const));
+    auto tail_shape = context.mark_node(std::make_shared<opset8::Broadcast>(one_const, tail_shape_rank));
+    auto channels_dim = context.mark_node(std::make_shared<opset8::ShapeOf>(bias));
+    auto new_shape =
+        context.mark_node(std::make_shared<opset8::Concat>(OutputVector{one_const, channels_dim, tail_shape}, 0));
+
+    return context.mark_node(std::make_shared<opset8::Reshape>(bias, new_shape, false));
+}
+
 OutputVector translate_convolution(NodeContext& context) {
     // Shchema: aten::_convolution(Tensor input, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[]
     // dilation, bool transposed, int[] output_padding, int groups, bool benchmark, bool deterministic, bool
     // cudnn_enabled, bool allow_tf32) -> Tensor
-    bool transposed = context.const_input<bool>(6);
-    // TODO: Handle this temporary limitation
-    OV_FRONTEND_REQUIRE(!transposed);
 
     auto strides = context.const_input<Strides>(3);
-    auto pads_begin = context.const_input<CoordinateDiff>(4);  // FIXME: The same 4 is used twice
-    auto pads_end = context.const_input<CoordinateDiff>(4);    // FIXME: The same 4 is used twice
+    auto pads = context.const_input<CoordinateDiff>(4);
     auto dilations = context.const_input<Strides>(5);
-    // TODO: Handle skipped input 7 (6 was used above) -- what is it for?
+    bool transposed = context.const_input<bool>(6);
+    auto output_padding = context.const_input<CoordinateDiff>(7);
     auto groups = context.const_input<int64_t>(8);
 
     std::shared_ptr<ov::Node> conv;
     if (groups == 1) {
-        conv = std::make_shared<opset8::Convolution>(context.get_input(0),
-                                                     context.get_input(1),
-                                                     strides,
-                                                     pads_begin,
-                                                     pads_end,
-                                                     dilations);
+        if (!transposed) {
+            conv = context.mark_node(std::make_shared<opset8::Convolution>(context.get_input(0),
```

Comment:
If `output_padding` different from 0 will be provided for `groups==1` and `transposed==False` case what will happen in torch?

Reply:
output_padding used only for transposed True, so it will be ignored